universal-login-sdk/src/core/models/WalletService.ts

```diff
@@ -26,6 +31,10 @@ export type WalletState = {
 
 export type SerializedWalletState = {
   kind: 'None';
+} | {
+  kind: 'Confirmed';
+  name: string;
```

Comment:
what is the purpose of this variable?

Reply:
I wasn't sure if we need it

Reply:
Should I remove it?

Reply:
i think yes. You have everything inside ConfirmedWallet. We will add it if we need it

Reply:
Done